src/includes/content/wizard/affliate/flycricket.pug

```diff
@@ -4,13 +4,12 @@
     #fc-callouts
         form#fc-form(action='https://app.flycricket.com/import/privacy' method='post' target='_blank' onsubmit='return fc_submit(event)')
             input(type='hidden' name='fc-referrer' value='nishant')
+            input#fc-terms(type='hidden' name='fc-terms')
             input#fc-body(type='hidden' name='fc-body')
             input#fc-name(type='hidden' name='fc-name' v-model='appName')
-            div(style='margin-top: 1em;')
-                button.button.is-light.is-small
-                    | Learn More
-                button.button.is-black.is-small
-                    | Deploy
-        .fc-powered-by
+        div(style='margin-top: 1em;')
+            button.button.is-black.is-small(@click="fc_deploy()")
+                | Deploy
```

Comment:
The indentation is changed here. Indentation in a PUG file defines where the div would be inside a container. It needs to be where it was earlier.

Reply:
This was an intentional change. I don't want the button to trigger the form directly as a 'submit' button. I switched it to use the javascript click, so the form gets the data injection before submission. So this indentation change should remain for proper form submission